standalone/pruntime/enclave/src/storage_sync.rs

```diff
@@ -0,0 +1,34 @@
+use crate::light_validation::LightValidation;
+use enclave_api::storage_sync::{BlockValidator, Error as SyncError, Result};
+use parity_scale_codec::Decode;
+
+impl BlockValidator for LightValidation<chain::Runtime> {
+    fn submit_finalized_headers(
+        &mut self,
+        bridge_id: u64,
+        header: chain::Header,
+        ancestry_proof: sp_application_crypto::Vec<chain::Header>,
```

Comment:
Why is it `sp_application_crypto::Vec`? I think it's just `alloc::vec::Vec`

Reply:
Not sure why, just copy-pasted the origin code.

Reply:
Ah, I got it. Our enclave does not `extern crate alloc`, so the `rust-analyzer` did her best to find one for me.